SpreadCheetah.SourceGenerator/WorksheetRowGenerator.cs

```diff
@@ -383,6 +391,42 @@ private static Dictionary<CellStyle, int> GenerateCreateWorksheetRowDependencyIn
         return cellStyleToStyleIdIndex;
     }
 
+    private static void GenerateCellValueConvertersArray(StringBuilder sb, EquatableArray<RowType> rowTypes)
+    {
+        if (!rowTypes.Any(type => type.Properties.Any(property => property.CellValueConverter.HasValue)))
+        {
+            return;
+        }
+        
+        sb.AppendLine(FormattableString.Invariant($$"""
+                               private static Dictionary<string, object> _cellValueConverters = new Dictionary<string, object>
```

Comment:
This seems to work correctly, but it has the disadvantage of a dictionary lookup per cell, and a cast from object to get the correct type. This can potentially affect performance in a negative way.

Perhaps we should instead consider using a static field per converter type. For example:
```csharp
private static readonly StringValueConverter _valueConverter0 = new StringValueConverter();
private static readonly NullableIntValueConverter _valueConverter1 = new NullableIntValueConverter();
private static readonly DecimalValueConverter _valueConverter2 = new DecimalValueConverter();
```
Using name `_valueConverter` plus a number can avoid potential naming collisions. The source generator would need to keep track of which field corresponds to which converter. This can be done with a Dictionary, similar to what `GenerateCreateWorksheetRowDependencyInfo` is doing.

Reply:
yes, you're right  thanks